### ISSUE TITLE
Update playground's `README`

### ISSUE THREAD

**Issue by mdubus** [meilisearch/meilisearch-rails#155]
Currently, the playground's `README` looks like this: https://github.com/meilisearch/meilisearch-rails/blob/main/playground/README.md

We do have [a section](https://github.com/meilisearch/meilisearch-rails/blob/main/CONTRIBUTING.md#playground-) in the `CONTRIBUTING` to help us install it, but it is not sufficient enough to understand how it works and how to make it run.

We should consider adding more explanations inside the `README` of the playground and add a link to it in the `CONTRIBUTING` in order to help new contributors understand how they can easily test their changes 🙃

**Comment by curquiza** [comment]
Hello @mdubus 

> it is not sufficient enough to understand how it works and how to make it run.

What do you mean exactly? So that we can improve the README accordingly to your expectations

**Comment by mdubus** [comment]
It may be my lack of knowledge of the rails environment, but here are the 2 main "problems" I encountered:
- By running `bundle exec rails webpacker:install` it asked me `Overwrite /Users/morgane/PROJECTS/INTEGRATION/meilisearch-rails/playground/babel.config.js?` and I didn't know what to do
- The `CONTRIBUTING.md` says:
```bash
bundle install
bundle exec rails db:create
bundle exec rails db:migrate
bundle exec rails db:seed
bundle exec rails webpacker:install
bundle exec rails server
```
But I don't know what this commands are for. I would have liked small comments to explain it briefly.

Other small "problems" I want to highlight:
- The `CONTRIBUTING.md` doesn't tell us that something will run on http://127.0.0.1:3000/ nor what the aim of this playground is. it is clear once running it, but by reading the `CONTRIBUTING.md` I didn't know what to expect from it.
- I first went to the playground's README to know how to start it, by instinct, but the content I was searching for was in the `CONTRIBUTING.md`, which confused me. If there is a README in the playground, I expect to find the explanations about how to start the playground directly in it.
